### ISSUE TITLE
Accordion - Incorrect triggering of Events: itemShown, itemHidden

### ISSUE THREAD

**Issue by Kiaba** [mdbootstrap/mdb-angular-ui-kit#156]
<img width="1238" alt="image" src="https://user-images.githubusercontent.com/11596015/189080258-89500bb9-42f8-4f98-adc9-9ce4c0762d5a.png">

File
[mdb-angular-ui-kit](https://github.com/mdbootstrap/mdb-angular-ui-kit)/[projects](https://github.com/mdbootstrap/mdb-angular-ui-kit/tree/master/projects)/[mdb-angular-ui-kit](https://github.com/mdbootstrap/mdb-angular-ui-kit/tree/master/projects/mdb-angular-ui-kit)/[accordion](https://github.com/mdbootstrap/mdb-angular-ui-kit/tree/master/projects/mdb-angular-ui-kit/accordion)/[accordion-item.component.html](https://github.com/mdbootstrap/mdb-angular-ui-kit/tree/master/projects/mdb-angular-ui-kit/accordion/accordion-item.component.html)

Instead of
```
(collapseShow)="onShown()"
(collapseHide)="onHidden()"
```

We have to use
```
(collapseShown)="onShown()"
(collapseHidden)="onHidden()"
```

**Comment by idzark** [comment]
Thank you for reporting this problem. I added this to our to-do list and it should be fixed in the next update.

**Comment by idzark** [comment]
This problem was fixed in v3.0.1
